commands/cfconfig/eventgatewayinstance/delete.cfc

```diff
@@ -0,0 +1,72 @@
+/**
+* Delete a event gateway instance. Identify the event gateway instance uniquely by the id.
```

Comment:
Just checking, is the gateway instance ID something that's visible in the web admin so that a user could copy paste it to use here?

Reply:
It's in the column **Gateway ID**
![image](https://user-images.githubusercontent.com/4364969/44834639-01013a00-ac32-11e8-9171-7485a41179c0.png)